specification/automation/resource-manager/Microsoft.Automation/preview/2023-05-15-preview/account.json

```diff
@@ -944,6 +1033,46 @@
         }
       }
     },
+    "DeletedRunbookProperties": {
+      "type": "object",
+      "properties": {
+        "runbookId": {
+          "type": "string",
+          "description": "Gets or sets the Runbook Id."
+        },
+        "runbookType": {
+          "type": "string",
+          "description": "Type of the runbook."
+        },
+        "runtime": {
+          "type": "string",
+          "description": "Gets or sets runtime of the runbook.",
+          "x-nullable": true
+        },
+        "runtimeEnvironment": {
+          "type": "string",
+          "description": "Environment of the runbook.",
+          "x-nullable": true
+        },
+        "creationTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the creation time.",
+          "x-nullable": false
+        },
+        "deletionTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the last modified time.",
+          "x-nullable": false
+        },
+        "description": {
```

Comment:
why do we need this?

Reply:
Not really essential, just in case if needed for differentiation from customer side